app/frontend/src/components/LabelAndText.tsx

```diff
@@ -0,0 +1,33 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+import TextBody from "./TextBody";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    marginTop: theme.spacing(0.5),
+  },
+  label: {
+    margin: "inherit",
```

Comment:
Is this to give the same margin as root?

Reply:
Yep, otherwise it's misaligned